internal/tags/key_value_tags.go

```diff
@@ -735,7 +735,11 @@ func (tags KeyValueTags) ResolveDuplicates(ctx context.Context, defaultConfig *D
 
 	result := make(map[string]string)
 	for k, v := range t {
-		result[k] = v.ValueString()
+		if v != nil {
```

Comment:
@acwwat it might be worth adding this `nil` check into the `ValueString()` method itself.

Reply:
Updated, thanks.